org_status/export/Exporter.py

```diff
@@ -0,0 +1,15 @@
+class Exporter:
+    NAME = None
+    OUTPUT_FILE = None
```

Comment:
This is just an encoder class, I am not sure why you need this. This class should not write anything to a file. This issue does not relate to making the tool work with any specific tool.